### ISSUE TITLE
Is there a way or model to remove the song(including lyrics) but keep speech?

### ISSUE THREAD

**Issue by SetoKaiba** [Anjok07/ultimatevocalremovergui#968]
Is there a way or model to remove the song(including lyrics) but keep speech?

**Comment by complexinteractive** [comment]
No. If this outcome is important to you, the best option is to layer the extracted instrumental and acapella tracks in a DAW and remove the vocal sections while leaving the speech in.
